src/client/globalsettings.rs

```diff
@@ -1,24 +1,19 @@
+use crate::client::output::outputs::Output;
 use std::path::{Path, PathBuf};
 
-use cli_table::ColorChoice;
-
 pub struct GlobalSettings {
-    color_policy: ColorChoice,
     server_dir: PathBuf,
+    pub printer: Box<dyn Output>,
```

Comment:
Please keep it private and create a getter (like `pub fn printer(&self) -> &dyn Output`). You can create a getter using `Alt + Insert` on the struct in IntelliJ.